src/api/controllers/db-controller.ts

```diff
@@ -1000,6 +1008,11 @@ function parseContractCallMetadata(
   if (!contract.found) {
     throw new Error(`Failed to lookup smart contract by ID ${parsedTx.contract_call.contract_id}`);
   }
+  if (!contract.result.abi) {
+    throw new Error(
```

Comment:
What happens if the contract is found but missing an ABI? Won't this cause the same error that the PR is intending to fix?

Reply:
Another misunderstanding. I am considering contracts without `ABI` to be faulty. Thus, throwing error on this point. 
I will return `parsedTx` at this point to return other information (i.e. without `ABI`)